src/server.py

```diff
@@ -68,15 +71,22 @@ def fetch_by_newspaper(url):
         "title": article.title,
         "topimage": article.top_image}), 200, {'Content-Type': 'application/json'}
 
+
 def is_youtube_url(url):
     for youtube_url in YOUTUBE_URLS:
         if url.startswith(youtube_url):
             return True
     return False
 
-def fetch_og_tags(url):
-    title, description, imageUrl = web_preview(url, timeout=20)
 
+def fetch_og_tags_internal(url):
```

Comment:
I think there's nothing "internal" here to define a new function. In the original `fetch_og_tags`, put an `if` then `return web_preview()` would be cleaner.